Content.Server/Power/Pow3r/BatteryRampPegSolver.cs

```diff
@@ -165,15 +172,15 @@ private void UpdateNetwork(Network network, PowerState state, float frameTime)
                     if (!battery.Enabled || !battery.CanDischarge || battery.Paused)
                         continue;
 
-                    var scaledSpace = battery.CurrentStorage * frameTime;
+                    var scaledSpace = battery.CurrentStorage / frameTime;
                     var supplyCap = Math.Min(battery.MaxSupply,
                         battery.SupplyRampPosition + battery.SupplyRampTolerance);
                     var supplyAndPassthrough = supplyCap + battery.CurrentReceiving * battery.Efficiency;
 
-                    battery.AvailableSupply = Math.Min(battery.CurrentStorage, supplyAndPassthrough);
+                    battery.AvailableSupply = Math.Min(scaledSpace, supplyAndPassthrough);
                     battery.LoadingNetworkDemand = unmet;
 
-                    battery.MaxEffectiveSupply = Math.Min(battery.CurrentStorage, battery.MaxSupply + battery.CurrentReceiving * battery.Efficiency);
+                    battery.MaxEffectiveSupply = Math.Min(battery.CurrentStorage / frameTime, battery.MaxSupply + battery.CurrentReceiving * battery.Efficiency);
```

Comment:
Isn't this just `scaledSpace`?

Reply:
Yes but that is actually how it was in the master code haha. I can change it but I just left it as is.